core/utils.ts

```diff
@@ -184,6 +184,18 @@ export function validate(compiledGraph: dataform.ICompiledGraph): dataform.IGrap
       validationErrors.push(dataform.ValidationError.create({ message, actionName }));
     }
 
+    // sqlserver config
+    if(!!action.sqldatawarehouse) {
```

Comment:
nit: this can just be `(action.sqldatawarehouse && action.sqldatawarehouse.distribution)`